twinkles/twinklesCatalogDefs.py

```diff
@@ -48,3 +48,37 @@ def get_phoSimMagNorm(self):
                 magNorm_out = magNorm + self.column_by_name(varName)
 
             return magNorm_out
+
+    def _query_and_write(self, filename, chunk_size=None, write_header=True,
```

Comment:
This does not look like it is any different from the ```_query_and_write()``` method defined in ```InstanceCatalog```.  Why did you reimplement it?